tokens/src/core/global/spacing.json

```diff
@@ -1,7 +1,7 @@
 {
   "spacing": {
     "grid": {
-      "gutter-width": { "value": "64px" }
+      "gutter-width": { "value": "32px" }
```

Comment:
@adamstankiewicz In a recent [pull request you changed](https://github.com/edx/brand-edx.org/commit/a29316d55715989647c0e741927625ba896ddb8f) the meaning of this design token. Unfortunately, your change affects the visual display of the [Layout component](https://github.com/openedx/paragon/issues/2520). Do you remember why you needed to change the value from `32px` to `64px`?